consensus/locking.md

```diff
@@ -0,0 +1,9 @@
+Coins locking
+==
+
+We want a predictable inflation so there MUST not be any token burning. In order
```

Comment:
How is this different than burning? Protocol mandated burning is visible and therefore easy to track.